test/add-channel.test.js

```diff
@@ -1,20 +1,21 @@
-const test = require('ava');
-const nock = require('nock');
-const {stub} = require('sinon');
-const proxyquire = require('proxyquire');
-const {authenticate} = require('./helpers/mock-github');
-const rateLimit = require('./helpers/rate-limit');
+import test from 'ava';
+import nock from 'nock';
+import sinon from 'sinon';
+import quibble from "quibble"
+
+import {authenticate} from './helpers/mock-github.js';
+import * as RATE_LIMIT from './helpers/rate-limit.js';
 
 /* eslint camelcase: ["error", {properties: "never"}] */
 
-const addChannel = proxyquire('../lib/add-channel', {
-  './get-client': proxyquire('../lib/get-client', {'./definitions/rate-limit': rateLimit}),
-});
+// mock rate limit imported via lib/get-client.js
+await quibble.esm('../lib/definitions/rate-limit.js', RATE_LIMIT) // eslint-disable-line
```

Comment:
i think this is an ok step for this round, but we may want to consider [testdouble](https://www.npmjs.com/package/testdouble), which uses `quibble` internally. that way, i think consolidating the use of `sinon` into just using a single library.

Reply:
Happy to look into that later. Or open it up as an issue for community contributors?

Reply:
I'll make a follow up issue for this, seems like a good candidate for the community to help with